src/zjs_i2c.json

```diff
@@ -0,0 +1,12 @@
+{
+    "module": "i2c",
+    "require": "i2c",
+    "depends": ["buffer", "ipm"],
+    "zephyr_conf": {
+        "all": [
+            "CONFIG_I2C=y"
+        ]
+    },
+    "zjs_config": ["-DBUILD_MODULE_I2C"],
+    "src": ["zjs_i2c.c", "zjs_i2c_ipm_handler.c"]
+}
```

Comment:
Needs the following added I think...
"src": ["zjs_i2c.c", "zjs_i2c_handler.c", "zjs_i2c_ipm_handler.c"]
"header": ["zjs_i2c.h", "zjs_i2c_handler.h", "zjs_i2c_ipm_handler.h"],
"init" : [zjs_i2c_init],
"cleanup" : ["zjs_i2c_cleanup"]